photon-core/src/main/java/org/photonvision/common/util/TimedTaskManager.java

```diff
@@ -45,8 +45,34 @@ public Thread newThread(@NotNull Runnable r) {
         }
     }
 
-    private final ScheduledExecutorService timedTaskExecutorPool =
-            new ScheduledThreadPoolExecutor(2, new CaughtThreadFactory());
+    private final ScheduledThreadPoolExecutor timedTaskExecutorPool =
```

Comment:
What does this change do?

Reply:
kinda looks like this is causing the builds to fail

Reply:
It's so I can get stack traces from timed tasks that throw an exception. We could instead wrap every lambda we get in our own try catch that might be less cursed?